wire-desktop-beta/PKGBUILD

```diff
@@ -1,57 +1,62 @@
 # Maintainer: Conor Anderson <conor@conr.ca>
-# Maintainer: Maxim Baz <$_pkgname at maximbaz dot com>
+# Maintainer: Maxim Baz <$pkgname at maximbaz dot com>
 
 pkgname=wire-desktop-beta
 _pkgname=${pkgname%-beta}
 pkgver=3.5.2881
-pkgrel=1
-pkgdesc='Modern, private messenger. Based on Electron.'
-arch=('x86_64' 'i686')
+pkgrel=2
+pkgdesc='End-to-end encrypted messenger with file sharing, voice calls and video conferences'
+arch=('x86_64')
 url='https://wire.com/'
 license=('GPL3')
-conflicts=('wire-desktop-bin' 'wire-desktop' 'wire-desktop-git')
-depends=('alsa-lib' 'gconf' 'gtk2' 'libxss' 'libxtst' 'nss' 'xdg-utils')
-makedepends=('cargo' 'npm' 'python2' 'git' 'yarn')
-optdepends=('hunspell-en: for English spellcheck support'
-            'noto-fonts-emoji: for colorful emoji made by Google'
-            'ttf-emojione: for colorful emoji made by EmojiOne')
 provides=('wire-desktop')
-source=("${pkgver}.tar.gz::https://github.com/wireapp/wire-desktop/archive/linux/${pkgver}.tar.gz"
-        "${_pkgname}.desktop")
+depends=('electron' 'xdg-utils')
+makedepends=('cargo' 'npm' 'python2' 'git' 'yarn')
+optdepends=('hunspell-en_US: for English spellcheck support'
+            'emoji-font: colorful emoji')
+source=("${pkgname}-${pkgver}.tar.gz::https://github.com/wireapp/${_pkgname}/archive/linux/${pkgver}.tar.gz"
+        "${pkgname}.desktop")
 sha256sums=('4216cd9c3a2c4920aec2f3c967181b04bfafdb1b47e526a8e823911cce704da1'
-            'cc9056cecff2aa49a9ce9c8376d57ec8c7c2cb8174f7966b5cdccbeb2e3751ea')
+            '79505a8ae908a26f74af3d72c24cb09815dc5688eca26ee4bbf57362a865d7c5')
+
+prepare() {
+  # Create launcher script
+  cat << EOF > "${pkgname}"
+#!/bin/sh
+
+electron "/usr/lib/${pkgname}"
```

Comment:
moved this back to PKGBUILD because it's nice to be able to use `${pkgname}` to generate the launcher script.